core/src/main/java/nl/nn/adapterframework/webcontrol/api/Webservices.java

```diff
@@ -212,6 +212,31 @@ public void write(OutputStream out) throws IOException, WebApplicationException
 		}
 	}
 
+	private String getServiceEndpoint(IAdapter adapter) {
+		String endpoint = "external address of ibis";
+		Iterator it = adapter.getReceivers().iterator();
+		while(it.hasNext()) {
+			IListener listener = ((Receiver) it.next()).getListener();
+			if(listener instanceof WebServiceListener) {
+				String address = ((WebServiceListener) listener).getAddress();
+				String serviceNameSpaceUri = ((WebServiceListener) listener).getServiceNamespaceURI();
+				if(StringUtils.isNotEmpty(address)) {
+					endpoint = address;
+				} else if(StringUtils.isNotEmpty(serviceNameSpaceUri)) {
+					endpoint=serviceNameSpaceUri;
+				} else {
+					endpoint=listener.getName();
+				}
```

Comment:
Routing based on the namespace uri doesn't mean the uri is the external address.
If no address has been provided, the endpoint should be `services/rpcrouter`.
See NamespaceUriProvider and NamespaceUriProviderBean